invenio_records_lom/resources/config.py

```diff
@@ -7,23 +7,27 @@
 
 """REST API configuration."""
 
+from flask_resources import ResponseHandler
 from invenio_records_resources.resources import RecordResourceConfig
 
+from .serializers import LOMUIJSONSerializer
+
+record_serializer = {
```

Comment:
nitpick: names for dicts are usually in plural even if they contain just one element (i.e. record_serializer***s***)
looks a bit strange when comparing with everyone else having the plural there...

(also looks strange in this line below: `response_handlers = record_serializer`, which assigns a singular to a plural...)

Reply:
do you mean usually in the context of invenio or in the context of python?

for me it is strange to see that an object has a plural name. Plural should be used for a list. e.g. https://stackoverflow.com/a/9201009

Reply:
Usually, as in: Even in this PR most added dictionaries are named in plural.

Singular seems to be used quite a bit too, though. Some thoughts on this:
- I did a quick grep through invenio-modules and plural is more common there.
- The pattern seems to be that while dictionaries *can* be objects, usually they are lists keyed by strings (I guess objects are often instances of classes?).
- The python standard library does categorize mappings (dicts are mappings) as a kind of collection of things ([link](https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes)).
- The type of a dict would usually be something like `dict[str, Type]`, which is a collection of `Type`s keyed by `str`s.

Frankly, what confused me about this variable-name specifically is that invenio-rdm has the same thing named in plural ([link](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/resources/config.py#L54)).